docs/contactus.rst

```diff
@@ -15,24 +15,15 @@ for SUMO community discussions.
 Kitsune hackers
 ===============
 
-If you're hacking on the Kitsune code and have questions, ping us on
-IRC.
+If you're hacking on the Kitsune code and have questions, ping us on `Matrix <https://wiki.mozilla.org/Matrix>`_.
 
-We hang out in ``#sumodev`` on irc.mozilla.org. We're usually around
-during the day in Eastern Time.
+We hang out in `#sumodev:mozilla.org <https://chat.mozilla.org/#/room/#sumodev:mozilla.org>`_.
+We're usually around during the day in Europe.
 
 If you ask something and all you get is silence, then it's probably
 the case that we're not around. Please try pinging us again.
 
-You can also use the `dev-sumo mailing list
-<https://www.mozilla.org/about/forums/#dev-sumo>`_.
+Current developers:
```

Comment:
Let's link to AUTHORS and update that if needed.

Reply:
AUTHORS got [renamed to CONTRIBUTORS](https://github.com/mozilla/kitsune/commit/2ba4b1573bd97f1e077980ac6f114fbef4e80c1e#diff-98f07bc20cb66328be238119df96c490) which was then later replaced with this: https://github.com/mozilla/kitsune/blob/master/CONTRIBUTORS.rst

So I would say CONTRIBUTORS serves a different purpose from this, CONTRIBUTORS is about attribution, this is about who a contributor with questions should contact.